common/helpers/field.js

```diff
@@ -246,14 +212,99 @@ function mapPersonToOption(person) {
   }
 }
 
+function appendDependent(questions, assessmentCategory, field, key) {
+  const question = find(questions, { key })
+  let dependent = {}
+
+  if (question) {
+    if (field.explicit) {
+      dependent = {
+        field: `${question.key}__yesno`,
+        value: 'yes',
+      }
+    } else {
+      dependent = {
+        field: assessmentCategory,
+        value: question.id,
+      }
+    }
+
+    return {
+      ...field,
+      dependent,
+    }
+  }
+
+  return field
+}
+
+function extractItemsForImplicitFields(fields, question) {
+  const key = question.key
+  const field = fields[key]
+  return fields[key] && !field.explicit
+}
+
+function decorateWithExplicitFields(questions, collection, field, key) {
+  const question = find(questions, { key })
+
+  if (question && field.explicit) {
+    const explicitKey = `${key}__yesno`
+    const explicitField = explicitYesNo(explicitKey)
+    explicitField.items[0].conditional = key
+    collection[explicitKey] = explicitField
+  }
+}
+function explicitYesNo(name) {
```

Comment:
I think this should still remain in the fields file as this isn't doing any logic. It's a skeleton of the field we want to create.

I think after this, we can get this change in for now. We may need to refactor again to cater for new journeys with Prison -> Court and Prison -> Prison too.

Reply:
`explicitYesNo` is called by a func that is in `field.js` -- I reckon it's less awkward to leave it in the same file instead of adding cross-calls.

Reply:
Ok, done.